src/actions/query.js

```diff
@@ -585,6 +592,7 @@ export function updateQuery(
 						&& meta
 						&& typeof meta.enableAI === 'boolean'
 					? {
+						...meta,
```

Comment:
@SavvyShah Explain the thought behind this

Reply:
If other options are present with enableAI inside of `meta`, then before we just discarded them? eg. `theme` property would be discarded if enableAI is passed as true.
```
{
  theme: "dark"
  enableAI: true
}
```

Can you explain why we were doing this?

Reply:
@SavvyShah The point is relevance, the theme property for instance isn't relevant for the above LOC and if you see the LOC is just concerned with the type of component calling the `executeQuery` method to conditionally call the AI endpoint.

Reply:
That was just an example. I am telling there would be other properties other than just `enablAI` which would be discarded. In this case it is `imageValue` which is relevant. The `meta` property exists because we want to pass additional props to the query. So the point of props not being relevant is meaningless.